src/build_workflow/build_args.py

```diff
@@ -10,10 +10,22 @@
 
 
 class BuildArgs:
+    SUPPORTED_PLATFORMS = [
+        "linux"
```

Comment:
We do also support `darwin` since https://github.com/opensearch-project/opensearch-build/pull/748, let's make sure we have a test so we don't remove it.